src/Behat/Behat/Tester/ServiceContainer/TesterExtension.php

```diff
@@ -81,8 +81,6 @@ public function configure(ArrayNodeDefinition $builder)
                             ? sys_get_temp_dir() . DIRECTORY_SEPARATOR . 'behat_rerun_cache'
                             : null
                     )
-                ->end()
```

Comment:
These `end()` calls are not really needed because they just return a value which is not used and they return a type which cannot be chained with another end call, so I just removed them